index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for parity-pmr 0.1
+// Project: https://github.com/paritytrading/node-parity-pmr#readme
+// Definitions by: Leo Vujanić <https://github.com/leovujanic>
+
+import {Buffer} from "node";
+
+/**
+ * Declares Parity PMR message structure
+ * Full reference can be found here https://github.com/paritytrading/parity/blob/master/libraries/net/doc/PMR.md
+ */
+export interface PMRMessage {
+    messageType: string;
+    version?: number;
+    timestamp?: number;
+    username?: string;
+    orderNumber?: string;
+    side?: string;
+    instrument?: string;
+    quantity?: number;
+    price?: number;
+    canceledQuantity?: number;
+    matchNumber?: number;
+    restingOrderNumber?: number;
+    incomingOrderNumber?: number;
+}
```

Comment:
Would it make sense to declare `PMRMessage` as a union of `PMRVersion`, `PMROrderEntered`, `PMROrderAdded`, `PMROrderCanceled`, and `PMRTrade`?